bot/handlers/start_handler.py

```diff
@@ -4,9 +4,42 @@
 
 async def start_handler(update: Update, context: CallbackContext):
     """Handle the /start command."""
-    chat_id = update.effective_chat.id
+    chat_id = update.effective_chat.id  # обєкт.об'єкт(або інше).елемент.
     name = update.effective_user.first_name
+    user_id = update.effective_user.id
+    channels = [
+        ["Test channel 1", "httpst.metask_test_channel1", -1001805771484],
+        ["Test channel 2", "httpst.metask_test_channel2", -1001763595562],
+        ["Test channel 3", "httpst.metask_test_channel3", -1001626957248],
+        ["Test channel 4", "httpst.metask_test_channel4", -1001800227165],
+    ]
+    print(update.message.chat.type)
 
-    message = f"Вітання {name}! Welcome to the Telegive bot. Ти живий а значить переміг."
+    print("in start handler")
 
-    await context.bot.send_message(chat_id=chat_id, text=message)
+    is_all_channels_member = await check_is_all_channels_member(channels, user_id,
+                                                                context.bot)  # Виклик асинхронної функції через `Await`, та данні які ми передаємо у функцію
+    print(is_all_channels_member)
+    if is_all_channels_member:
+        message = f"Вітання {name}! Welcome to the Telegive bot. Ти підписаний а значить переміг."
+        await my_send_message(chat_id, message, context.bot)
+    else:
+        await my_send_message(chat_id, "не підписаний на всі канали ", context.bot)
+
+
+async def my_send_message(chat_id, text, bot):
+    print("in send message")
+
+    await bot.send_message(chat_id=chat_id, text=text)
```

Comment:
I think we don't need this `send_message` wrapper anymore, you can remove it and call `send_message` directly from bot.